Containers/mastercontainer/Caddyfile

```diff
@@ -21,7 +21,14 @@
 }
 
 http://:80 {
-  redir https://{host}{uri}
+    redir https://{host}{uri}
+}
+
+https://:8080 {
+    reverse_proxy localhost:8000
+    tls internal {
+        on_demand
```

Comment:
This needs double check if also ip-addresses get accepted here...

Reply:
yes

Reply:
should be done with https://github.com/nextcloud/all-in-one/pull/3416/commits/9a0fd6d0a21530a54cf196ab00ab7c94ee511164

Reply:
is this not a global option? https://caddyserver.com/docs/caddyfile/options#trusted-proxies

Reply:
I think it is both...

Reply:
https://caddyserver.com/docs/caddyfile/directives/reverse_proxy#the-http-transport